packages/common/core-utils/src/events/emitter.ts

```diff
@@ -40,14 +41,15 @@ export interface IEmitter<TListeners extends Listeners<TListeners>> {
 /**
  * Called when the last listener for `eventName` is removed.
  * Useful for determining when to clean up resources related to detecting when the event might occurs.
+ * @internal
  */
 export type NoListenersCallback<TListeners extends object> = (
 	eventName: keyof Listeners<TListeners>,
 ) => void;
 
 /**
  * Allows querying if an object has listeners.
- * @sealed
+ * @sealed @internal
```

Comment:
Not sure if it's required, but we typically put each of these on a separate line.
```suggestion
 * @sealed
 * @internal
```